.circleci/config.yml

```diff
@@ -1,46 +1,167 @@
----
-version: 2.0
+version: 2
 workflows:
   version: 2
   tests:
-    jobs:
-      - node8:
-          filters:
+    jobs: &workflow_jobs
+      - node6:
```

Comment:
The code was node8+ originally. Does changing to TypeScript make it compatible with Node 6?

Reply:
It does now :)